docs/mobile-apps/automated-testing/appium/appium-2-deque-drivers.md

```diff
@@ -0,0 +1,174 @@
+---
+id: appium-deque-drivers
+title: Deque axe DevTools® Mobile Analyzer Accessibility Drivers
+sidebar_label: Appium Deque axe DevTools Integration
+description: Learn how to run accessibility tests using axe DevTools® Mobile Analyzer drivers hosted in Sauce Labs' private cloud.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<p><span className="sauceGreen">Real Devices</span></p>
+
+[Deque's axe™](https://www.deque.com/axe/) is a leading digital accessibility toolkit. 
+The [axe DevTools® for Mobile SDK](https://docs.deque.com/devtools-mobile/appium) extends this functionality to mobile platforms, 
```

Comment:
We should make sure that the Axe documentation also mentions the minimum os version that we document here. 

We should also check that their documentation is up to date or if anything else needs to be changed there.

Reply:
will confirm this with Deque team

Reply:
I asked the team to update the docs accordingly.